src/components/Navbar/Navbar.jsx

```diff
@@ -1,27 +1,31 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
 import "./navbar.css";
 
-const Navbar = () => {
+export const Navbar = () => {
 	return (
 		<>
 			<header className="nav fixed">
 				<div className="menu-logo">
-					<div className="menu-btn">
-						<img
-							className="logo-svg"
-							src="/assets/logo/logo simple dark.svg"
-							alt="logo-svg"
-						/>
-					</div>
+					<NavLink to="/">
+						<div className="menu-btn">
+							<img
+								className="logo-svg"
+								src="/assets/logo/logo simple dark.svg"
```

Comment:
export all assets from a single file and use it, importing as `{ }` , wherever needed.